src/main/java/org/ibp/api/java/inventory/manager/LotTemplateExportService.java

```diff
@@ -1,12 +1,13 @@
 package org.ibp.api.java.inventory.manager;
 
-import org.ibp.api.domain.location.LocationDto;
+import org.generationcp.middleware.api.location.LocationDTO;
 import org.ibp.api.domain.ontology.VariableDetails;
 
 import java.io.File;
 import java.util.List;
 
 public interface LotTemplateExportService {
 
-	File export(final List<LocationDto> locations, final List<VariableDetails> scales);
+	File export(final List<LocationDTO> locations, final List<VariableDetails> scales);
```

Comment:
@lik-83 please take the chance to remove the finals

Reply:
done